src/assertions/OrderAssertions.ts

```diff
@@ -158,11 +156,23 @@ export class OrderAssertions {
     orders: (TakerWalletOrder | ZeroExSignedFillOrder)[],
   ) {
     let makerTokensUsed = SetProtocolUtils.CONSTANTS.ZERO;
+    const requiredComponentFills: { [addr: string]: BigNumber } = {};
```

Comment:
Use the type `Address` instead of string

Reply:
can't this has to be a primitive type, not an alias